Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -10862,14 +10862,16 @@ bool CStaticFunctionDefinitions::SendSyncIntervals(CPlayer* pPlayer)
     BitStream.pBitStream->Write(g_TickRateSettings.iKeySyncRotation);
     BitStream.pBitStream->Write(g_TickRateSettings.iKeySyncAnalogMove);
 
-    if (pPlayer->CanBitStream(eBitStreamVersion::FixSyncerDistance))
+    if (pPlayer)
     {
-        BitStream.pBitStream->Write(g_TickRateSettings.iPedSyncerDistance);
-        BitStream.pBitStream->Write(g_TickRateSettings.iUnoccupiedVehicleSyncerDistance);
-    }
```

Comment:
You need min client version check here.
The new client always expects all data.

Reply:
Well, thanks for reporting the issue, but could you please elaborate on it and how exactly it should be solved?